### ISSUE TITLE
Netbeans plugin

### ISSUE THREAD

**Issue by egraether** [CoatiSoftware/SourcetrailBugTracker#420]

**Comment by egraether** [comment]
+1 by mail

**Comment by diegodiasm** [comment]
Is anybody currently working on this issue? Is Netbeans expected to be of the IDEs supported by Sourcetrail?

**Comment by egraether** [comment]
No, there is currently nobody working on it. The demand is also low. Maybe you find some time to look into it. The protocol is simple and there are lots of other plugins for guideline. Please take a look at our [plugin documentation](https://www.sourcetrail.com/documentation/#CODEEDITORPLUGINS).

**Comment by mlangkabel** [comment]
A rather simple plugin that can be used as a reference implementation the the Sublime plugin. It is basically just [this single Python file](https://github.com/CoatiSoftware/sublime-sourcetrail/blob/master/SourcetrailPlugin.py).
